README.md

```diff
@@ -140,6 +139,25 @@ If we now add a Shell script with a simple echo from that value, we can see that
 
 ARM Deploy Actions is supported for the Azure public cloud as well as Azure government clouds ('AzureUSGovernment' or 'AzureChinaCloud') and Azure Stack ('AzureStack') Hub. Before running this action, login to the respective Azure Cloud  using [Azure Login](https://github.com/Azure/login) by setting appropriate value for the `environment` parameter.
 
+## Example on how to use failOnStdErr
+In this example, we are setting `failOnStdErr` to false. 
+
+```yaml
+- uses: azure/arm-deploy@v1
+  id: deploy
+  with:
+    resourceGroupName: azurearmaction
+    template: examples/template/template.json
+    parameters: examples/template/parameters.json
+    deploymentName: github-advanced-test
+    failOnStdErr: false
+```
+The default value of `failOnStdErr` is true to support back compatibility. `failOnStdErr` equals true means that if there is some data written to stdErr and return code from az-cli is 0 then we will fail the action. 
```

Comment:
Remove this `The default value of `failOnStdErr` is true to support back compatibility.`